src/include/quick-lint-js/error.h

```diff
@@ -145,6 +145,12 @@
       .error(u8"number literal contains trailing underscore",                  \
               underscores))                                                    \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_number_literal_contains_trailing_underscores,                      \
+      { source_code_span underscores; },                                       \
+      .error(u8"number literal contains trailing underscores",                 \
+              underscores))                                                    \
```

Comment:
Blocking: Let's not make two error types here. Two error types for basically the same thing makes the code more complicated IMO. Let's make only one error type: error_number_literal_contains_trailing_underscores

Reply:
Thank your for the review!

I modified the code so there is only one error type.
The new error type works for both "123456_" and "123456__".